app/models/user.rb

```diff
@@ -1,2 +1,10 @@
 class User < ApplicationRecord
+  # Associations
+  has_many :posts, dependent: :destroy, foreign_key: 'author_id'
+  has_many :comments, dependent: :destroy, foreign_key: 'author_id'
+  has_many :likes, dependent: :destroy, foreign_key: 'author_id'
+
+  def recents_post
+    posts.limit(3).order(created_at: :asc)
```

Comment:
- [ ] The recents_post method should list the last 3 posts, kindly make sure to change `order.(created_at: :asc)` to `order('created_at Desc')` so it returns the last 3 posts, as for now the method returns the 3 first posts 👇🏼 

![Screenshot_20230331_033249](https://user-images.githubusercontent.com/25227022/229016213-44a220c5-ca36-4a78-baf3-3b029ebdea46.png)